sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -70,6 +72,41 @@ internal class NextLinkOperationImplementation : IOperation
             return new OperationToOperationOfT<T>(operationSource, operation);
         }
 
+        public static IOperation Create(
```

Comment:
Lot's of parsing and assumed values here.   Where are the tests?

Reply:
Added UT